apps/dashboard/src/lib/components/Org/LandingPage/index.svelte

```diff
@@ -560,12 +561,15 @@
             {/if}
           </div>
         </ul>
-        <p class="text-center mt-5">
+        <!-- <p class="text-center mt-5">
           Powered by
           <a class="text-primary-600 underline" href="https://classroomio.com" target="_blank"
             >ClassroomIO</a
           >
-        </p>
+        </p> -->
+        <div class="flex items-center justify-center">
+          <Badge />
+        </div>
```

Comment:
@Chifez Please use the same positioning on this page. Keep it on the right like on the course landing page